REVIEW_PROCESS.md

```diff
@@ -0,0 +1,63 @@
+
+Reviewers are selected by the author. The author may select more than two reviewers, if desired, but only the approval of two reviewers is needed. The review *may* consist of in-line comments, but it *must* include the template. 
+
+Anybody who wishes to give feedback on the PR may do so, and the author is encouraged to consider all feedback thoughtfully, whether it comes from casual commenters or official reviewers. There is no deadline between initial submission of the PR and assignment of reviewers. If the author wishes, they may submit the PR, wait a while, see who comments on it, and then invite the commenters to become reviewers.
+
+There is no rule of thumb on how long the review needs to be. There is also no rule of thumb on how many comments the author must respond to. All that matters is that two reviewers must ultimately approve the article. Reviewers are encouraged to be reasonable and understand that not all of their comments will be heeded. 
+
+Reviewers have **three weeks** to complete their review. After three weeks, it is up to the authors’ discretion whether they wish to give the tardy reviewer more time, or select a new reviewer.
+
+If an article fails to achieve the approval of two reviewers in a reasonable time frame, the PR will be closed. (Who decides this, and how, will be played by ear.) The author may wish to re-submit a modified version of the article. Authors are discouraged from simply re-submitting the same article with more agreeable reviewers. In the event of disagreement on a data standard, the disagreeing parties are encouraged to meet—preferably over food and drink—and have one or more in-person discussions to try and achieve consensus. If consensus is simply impossible, the disagreeing parties should each write an article arguing for their position. The two articles should link to each other on the website, and should respond to each other’s ideas.
+
+---
+
+# Review template
+
+The questions below do not need to be answered line-by-line. Rather, they are meant to convey a general sense of the purpose of that section. It is up to the reviewers to decide how thoroughly they wish to address each section.
+
+## 1. The Big Picture
+
+Is this article appropriate for the Janelia Data Standards project? Is the general premise of the article sound? Is the proposed data standard generalizable beyond that author’s particular use case?
+
+## 2. Technical Choices
+
+Is the proposed standard elegant, straightforward, and focused? Does the author adequately explain the rationale behind the standard? Is the author making any flawed implicit assumptions, either about the problem or about their audience? Does the author provide an implementation and/or example data?
```

Comment:
It seems to me that all the questions listed here are necessary conditions for positively evaluation of an article, but most of them are up to interpretation of the reviewer. The only question that is pretty unambiguous is the one about implementation and/or example data. In this sense, this question is qualitatively different from all others asked here.
Do we explicitly want to require an implementation and/or example data? I'm not saying we should or shouldn't. I just want to point out that it's not completely clear to me if it's required in the current version.

Reply:
Changed this:

> [the questions] are meant to convey a general sense of the purpose of that section.

To this:

> [the questions] are meant to guide the reviewer in their subjective evaluation of the article's quality.

And added that example data/implementation are "preferred but not required".